aws/data_source_aws_imagebuilder_image_recipe.go

```diff
@@ -116,6 +116,10 @@ func dataSourceAwsImageBuilderImageRecipe() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"working_directory": {
```

Comment:
This new attribute needs documentation added in `websites/docs/d/imagebuilder_image_recipe.html.markdown` -- will followup on merge.